pkg/supervisor.go

```diff
@@ -305,6 +324,7 @@ func ParseCompose(config *compose.Config) ([]Service, error) {
 			Env:       env,
 			Mounts:    mounts,
 			DependsOn: s.DependsOn,
+			User:      s.User,
```

Comment:
There was no equivalent "group" in the compose spec to import, so I've left that off for now. It wasn't required for the fix.

Reply:
Per the docs I found, the user value in the compose file takes the same values as the `docker run` command https://docs.docker.com/engine/reference/run/#user  so you can specify group via the `user:group` notation, which containerd also accepts, so I don't think anything needs to be done. But it might be worth documenting incase it is needed in the future. it will be a will reduce a lot of frustration to have a small note here

Reply:
👍

Reply:
Added a note.